webpack.config.js

```diff
@@ -93,6 +94,8 @@ module.exports = {
       filename: 'actions.html',
       title: 'Blockstack',
       chunks: ['actions'],
+      segmentHost: 'cdn.segment.io',
```

Comment:
Instead of using them here, let's just add this to `DefinePlugin`. That way, we won't have to repeat this for each HTML target.